src/cli.ts

```diff
@@ -109,7 +109,7 @@ try {
         flag: argv.force ? 'w' : 'wx',
     });
 } catch (e) {
-    if (e.code == 'EEXIST') {
+    if (typeof e === 'object' && e != null && (e as any).code === 'EEXIST') {
```

Comment:
Can you share more about the difference between this approach and using `instance of Error`? Your [linked article](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#use-unknown-catch-variables) had one example of that. Does this approach work in a scenario where `instanceof Error` would not?

Reply:
[This discussion](https://github.com/facebook/jest/issues/10784#issuecomment-989217138) we were looking at the other day spooked me a bit from using that sort of type check, since some of our samples using this library do involve invoking it from the context of a node jest environment.